pages/views/search.njk

```diff
@@ -6,6 +6,6 @@ eleventyExcludeFromCollections: true
 collectionIcon: sidebar/search.svg
 ---
 <esl-d-search-page-wrapper>
-  <div class="animation-loading bg-blue m-auto"></div>
-  <div class="gcse-search" data-resultsUrl="{{ '/search.html' | url }}"></div>
+   <div class="animation-loading bg-blue m-auto"></div>
```

Comment:
As far as it is looked, it is incorrect there should be a two spaces instead of 3